File: Dataset/Leetcode/valid/12/532.java
 class Solution {
    public String XXX(int num) {
        StringBuilder ans = new StringBuilder("");
        while (num>0){
            if (num>=1000){
                ans.append("M");
                num -= 1000;
            }else if (num>=900){
                ans.append("CM");
                num -= 900;
            }else if (num>=500){
                ans.append("D");
                num -= 500;
            }else if (num>=400){
                ans.append("CD");
                num -= 400;
            }else if (num>=100){
                ans.append("C");
                num -= 100;
            }else if (num>=90){
                ans.append("XC");
                num -= 90;
            }else if (num>=50){
                ans.append("L");
                num -= 50;
            }else if (num>=40){
                ans.append("XL");
                num -= 40;
            }else if (num>=10){
                ans.append("X");
                num -= 10;
            }else if (num>=9){
                ans.append("IX");
                num -= 9;
            }else if (num>=5){
                ans.append("V");
                num -= 5;
            }else if (num>=4){
                ans.append("IV");
                num -= 4;
            }else if (num>=1){
                ans.append("I");
                --num;
            }
        }
        return ans.toString();
    }
}

File: Dataset/Leetcode/test/118/532.cpp
 class Solution {
public:
    vector<vector<int>> XXX(int numRows) {
        vector<vector<int>> ans;
        for(int i = 1; i <= numRows; ++i) {
            vector<int> dump {1};
            while(dump.size() < i) {
                int a = dump.size() < ans.back().size() ? ans.back()[dump.size()] : 0;
                dump.push_back(a+ans.back()[dump.size()-1]);
            }
            ans.push_back(std::move(dump));
        }
        return ans;
    }
};

